index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title></title>
+  <link rel="stylesheet" href="stylesheets/style.css">
+  <link rel="stylesheet" href="stylesheets/index.css">
+</head>
+<body>
+  <header>
+    <div id="header">
+      <h1>The ADA Hall of Fame</h1>
+    </div>
+    <div>
+      <ul class="main_page_links">
+          <li><a href="./pages/education.html">Education</a></li>
+          <li><a href="./pages/hobbies.html">Hobbies</a></li>
+          <li><a href="./pages/fun_facts.html">Fun Facts</a></li>
+      </ul>
+    </div>
+    
+  </header>
+  <main>
+    <section class="instructors">
+      <nav class="nav_adrian">
+        <img src="images/Adrian-Prado-600x600.png" alt="Hey, Adrian!" width="300" height="200">
+        <h3>Adrian Prado</h3>
+        <h3>(he/him)</h3>
+        <h4>Instructor</h4>
+      </nav>
+      <nav class="nav_ansel">
+        <img src="images/Ansel-300x300.png" alt="Hey, Ansel!" width="300" height="200">
+        <h3>Ansel Rognile</h3>
+        <h3>(he/him)</h3>
+        <h4>Instructor</h4>
+      </nav>
+      <nav class="nav_ashley">
+        <img src="images/Ashley.png" alt="Hey, Ashley!" width="300" height="200">
+        <h3>Ashley Yang</h3>
+        <h3>(she/her)</h3>
+        <h4>Instructor</h4>
+      </nav>
+      <nav class="nav_kelsey">
+        <img src="images/Kelsey-Steven-600x600.png" alt="Hey, Kelsey!" width="300" height="200">
+        <h3>Kelsey Steven</h3>
+        <h3>(they/them)</h3>
+        <h4>Instructor</h4>
+      </nav>
+      <nav class="nav_ada">
+        <img src="images/ada-icon.png" alt="Ada Icon" width="300" height="200">
```

Comment:
We want to make sure that we have more descriptive `alt` tags, especially for users who are using screen readers.